client/layout/guided-tours/tours/checklist-contact-page-tour.js

```diff
@@ -108,8 +113,8 @@ export const ChecklistContactPageTour = makeTour(
 			style={ { marginTop: '-10px' } }
 		>
 			<Continue target="editor-publish-button" step="finish" click>
-				{ translate( 'Almost done, press the {{b}}Update{{/b}} button to save your changes.', {
-					components: { b: <strong /> },
+				{ translate( 'Almost done, press the {{updateButton/}} button to save your changes.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Almost done, press the {{UpdateButton/}} button to save your changes.' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).